### ISSUE TITLE
M3 does not package example projects

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-eclipse#907]

**Comment by cdietrich** [comment]
on master this is packaged. wonder how this can have happened

**Comment by cdietrich** [comment]
even here it is in the jar: https://ci.eclipse.org/xtext/job/xtext-eclipse/job/milestone_2.16.0.M3/2/execution/node/3/ws/build/p2-repository/plugins/

@kthoms why is it not in the update site?
does the job at eclipse do something special with the jar?

**Comment by cdietrich** [comment]
at https://ci.eclipse.org/xtext/job/xtext-umbrella/job/milestone_2.16.0.M3/4/execution/node/3/ws/build/p2-repository/plugins/ its still there

**Comment by cdietrich** [comment]
i really wonder where the broken jar was picked from.
we have to watch this duplicate and triple on release

**Comment by cdietrich** [comment]
https://ci.eclipse.org/xtext//job/xtext-eclipse/job/milestone_2.16.0.M3/lastStableBuild/artifact/build/p2-repository/plugins/

contents folder has size 0

**Comment by cdietrich** [comment]
```
INFO] 
[INFO] --- tycho-packaging-plugin:1.2.0:package-plugin (default-package-plugin) @ org.eclipse.xtext.xtext.ui.examples ---
[INFO] Building jar: /jobs/genie.xtext/xtext-eclipse/branches/milestone-2-16-0-M3.3t131m/workspace/org.eclipse.xtext.xtext.ui.examples/target/org.eclipse.xtext.xtext.ui.examples-2.16.0-SNAPSHOT.jar
[INFO] 
[INFO] --- tycho-p2-plugin:1.2.0:p2-metadata-default (default-p2-metadata-default) @ org.eclipse.xtext.xtext.ui.examples ---
[INFO] 
[INFO] --- maven-antrun-plugin:1.8:run (default) @ org.eclipse.xtext.xtext.ui.examples ---
[INFO] Executing tasks

main:
```
the order looks bogus. shouldn't antrun run first?

mvn clean does not delete
/org.eclipse.xtext.xtext.ui.examples/contents

**Comment by LorenzoBettini** [comment]
probably it is not bound to the right phase? I'd suggest to bind the antrun to the "compile" phase

**Comment by cdietrich** [comment]
yes that is an additional problem. will move to generate-sources as xtend does it.

**Comment by cdietrich** [comment]
Fixed in 2.16
